plugin/storage/badger/factory.go

```diff
@@ -29,8 +29,10 @@ import (
 	"github.com/jaegertracing/jaeger/pkg/metrics"
 	"github.com/jaegertracing/jaeger/plugin"
 	depStore "github.com/jaegertracing/jaeger/plugin/storage/badger/dependencystore"
+	samplingStore "github.com/jaegertracing/jaeger/plugin/storage/badger/samplingstore"
```

Comment:
very confusing name given `samplingstore` package also imported.
```suggestion
	badgerSampling "github.com/jaegertracing/jaeger/plugin/storage/badger/samplingstore"
```